.github/workflows/ci.yml

```diff
@@ -0,0 +1,114 @@
+name: CI
+
+on: pull_request
+
+env:
+  BUNDLE_PATH: vendor/bundle
+  BUNDLE_JOBS: 4
```

Comment:
do we know how many cores github actions provide to us? Otherwise we could use something like `nproc` maybe?

Reply:
I don't know, but with proper caching it isn't that much of a factor.